pkg/apis/core/types_shoot.go

```diff
@@ -333,6 +333,8 @@ type ClusterAutoscaler struct {
 	Expander *ExpanderMode
 	// MaxNodeProvisionTime defines how long CA waits for node to be provisioned (default: 20 mins).
 	MaxNodeProvisionTime *metav1.Duration
+	// MaxGracefulTerminationSec number of seconds CA waits for pod termination when trying to scale down a node (default: 600).
```

Comment:
```suggestion
	// MaxGracefulTerminationSeconds is the number of seconds CA waits for pod termination when trying to scale down a node (default: 600).
```

Reply:
I was also wondering about this naming convention. Kept it this way as it matches with the CA flag name. However, I agree that `MaxGracefulTerminationSeconds` is easier to understand. I will make this change.

Reply:
Thanks!

Reply:
Made both suggested changes of renaming var and updating doc string with [this commit](https://github.com/gardener/gardener/pull/4697/commits/a849f80f77b6b70c2f7fc080c53122926e70b412)